skl2onnx/shape_calculators/sgd_oneclass_svm.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from ..common._registration import register_shape_calculator
+from ..common.data_types import Int64TensorType, FloatTensorType
+
+
+def calculate_sgd_oneclass_svm_output_shapes(operator):
+    N = operator.inputs[0].get_first_dimension()
+    operator.outputs[0].type = Int64TensorType([N, ])
+    operator.outputs[1].type.shape = [N, 1]
+    operator.outputs[1].type = FloatTensorType([N, ])
```

Comment:
Usually, we do not use FloatTensorType but the same type as operator.inputs. That way, the convertor works for double as well.

Reply:
got it and resolved.